server/lobbyconnection.py

```diff
@@ -909,14 +914,24 @@ async def command_game_host(self, message):
         mapname = message.get("mapname") or "scmp_007"
         password = message.get("password")
         game_mode = mod.lower()
+        rating_min = message.get("rating_min")
+        rating_max = message.get("rating_max")
+        enforce_rating_range = bool(message.get("enforce_rating_range", False))
+        if rating_min is not None:
+            rating_min = float(rating_min)
+        if rating_max is not None:
+            rating_max = float(rating_max)
```

Comment:
rating_min
rating_max
enforce_rating_range

are the new properties on the create_game message